nextflow.config

```diff
@@ -45,6 +45,27 @@ params {
     validationSchemaIgnoreParams     = 'genomes,igenomes_base'
     validationShowHiddenParams       = false
     validate_params                  = true
+
+    // Profile dists args
+    pd_outfmt = "matrix"
+    pd_distm = "scaled"
+    pd_missing_threshold = 1.0
+    pd_sample_quality_threshold = 1.0
+    pd_match_threshold = -1.0
+    pd_file_type = "text"
+    pd_mapping_file = null // default is no file
+    pd_force = false
+    pd_skip = false
+    pd_columns = null
+    pd_count_missing = true
+
+    // GAS Cluster
+    gm_thresholds = "10,5,0"
+    gm_delimiter = "'.'" // note the single quotes surrounding the delimiter
+
+    // Arborview specific data
+    // TODO check this works in azure
+    av_html = "./assets/ArborView.html"
```

Comment:
Given that this is a new pipeline, did we decide to put these parameters in `nextflow.config`, or should we be putting them in the `modules.config` before there's too much technical debt built up in `nextflow.config`?

Reply:
I think this might be worth discussing, as the parameters passed to ProfileDists `pd_*` and GAS Mcluster `gm_*` are meant to be set from the top level. But I think the ArborView.html can be set in the `modules.config`

Reply:
I did not adjust this, but I did not forget about it! Ill make the changes in a new PR